.golangci.yml

```diff
@@ -0,0 +1,23 @@
+linters:
```

Comment:
FYI this is the config we've been using elsewhere:
```
linters:
  enable:
  - asciicheck
  - deadcode
  - depguard
  - errcheck
  - errorlint
  - gofmt
  - goimports
  - gosec
  - gocritic
  - importas
  - prealloc
  - revive
  - misspell
  - stylecheck
  - tparallel
  - unconvert
  - unparam
  - whitespace
output:
  uniq-by-line: false
issues:
  exclude-rules:
  - path: _test\.go
    linters:
    - errcheck
    - gosec
  max-issues-per-linter: 0
  max-same-issues: 0
run:
  issues-exit-code: 1
  timeout: 10m
```

Reply:
hum i will update rekor with that as well :)

Reply:
thanks for the review Matt